backend/src/main/java/com/soen341/instagram/security/WebSecurityConfig.java

```diff
@@ -0,0 +1,56 @@
+package com.soen341.instagram.security;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.authentication.AuthenticationManager;
+import org.springframework.security.config.annotation.authentication.builders.AuthenticationManagerBuilder;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+import org.springframework.security.core.userdetails.UserDetailsService;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import org.springframework.security.crypto.password.PasswordEncoder;
+
+@Configuration
+@EnableWebSecurity
+/*
+ * This class main purpose is to get the authentication manager bean. All HTTP
+ * request configuration should be done in
+ * com.soen341.instagram.security.ResourceServerConfig
+ */
+public class WebSecurityConfig extends WebSecurityConfigurerAdapter
+{
+	// This class main purpose is to get the authentication manager bean. All HTTP
```

Comment:
Not sure it's necessary to have this comment writte out twice!

Reply:
My mistake, didn't notice I repeated it twice. Going to remove it.